docs/atlascli/command/atlas-serverless-update.txt

```diff
@@ -81,6 +81,10 @@ Options
      - string
      - false
      - Hexadecimal string that identifies the project to use. This option overrides the settings in the configuration file or environment variable.
+   * - --tag
+     - stringToString
+     - false
+     - Collection of key-value pairs that categorizes the serverless instance. This will replace existing tags. This value defaults to [].
```

Comment:
```suggestion
     - List that contains key-value pairs between 1 to 255 characters in length for tagging and categorizing the serverless instance. This will replace existing tags. This value defaults to [].
```